_data/catalog/widgets.json

```diff
@@ -1167,6 +1167,17 @@
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
   {
+    "name": "RefreshIndicator",
+    "description": "A material design pull-to-refresh wrapper for scrollables.",
```

Comment:
Material Design

Reply:
replaced all 11 instances in the catalog